cmd/svcat/output/binding.go

```diff
@@ -75,33 +63,23 @@ func writeBindingListTable(w io.Writer, bindingList *v1beta1.ServiceBindingList)
 // WriteBindingList prints a list of bindings in the specified output format.
 func WriteBindingList(w io.Writer, outputFormat string, bindingList *v1beta1.ServiceBindingList) {
 	switch outputFormat {
-	case "json":
-		writeBindingListJSON(w, bindingList)
-	case "yaml":
-		writeBindingListYAML(w, bindingList)
-	case "table":
+	case formatJSON:
```

Comment:
This is sooooooooo much better! 

![excellent](https://user-images.githubusercontent.com/1368985/38935502-1cae98ae-42e4-11e8-9939-bd0474219157.jpg)